michael_lin/week_02/mcluhanWebsite/css/style1.css

```diff
@@ -0,0 +1,235 @@
+  height: 100vh;
+  -webkit-transform-style: preserve-3d;
+  transform-style: preserve-3d;
+}
+
+.parallax_layer {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
+.parallax_layer-fore {
+  -webkit-transform: translateZ(90px) scale(.7);
+  transform: translateZ(90px) scale(.7);
+  z-index: 1;
+}
+
+.parallax_layer-base {
+  -webkit-transform: translateZ(0);
+  transform: translateZ(0);
+  z-index: 4;
+}
+
+.parallax_layer-back {
+  -webkit-transform: translateZ(-300px) scale(2);
+  transform: translateZ(-300px) scale(2);
+  z-index: 3;
+}
+
+.title {
+  text-align: center;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+}
+
+
+#page1 {
+  z-index: 4;
+}
+#page1 .parallax_layer-base {
+  background: url(../images/mcluhan2.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+#page2 {
+  z-index: 2;
+}
+
+#page2 .parallax_layer {
+  background: url(../images/verticalgalaxy.jpg);
+  opacity: 0.85;
+  no-repeat: no-repeat;
+  background-size: cover;
+}
+
+#page2 .parallax_layer-back {
+  background: url(../images/mcluhan1.jpg);
+  no-repeat: no-repeat;
+  background-size: cover;
+}
+
+#page3 {
+  z-index: 3;
+}
+#page3 .wrapper {
+  margin: 0 auto;
+}
+
+#page3 .parallax_layer-base {
+  background: url(../images/mcluhan5.jpg);
+  background-size: cover;
+  vertical-align: bottom;
+}
+
+#page3 .parallax_layer-fore {
+  background: url(../images/tvwall.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  opacity: 0.2;
+
+}
+
+#page4 {
+  z-index: 1;
+  background: url(../images/mcluhan4.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+/* FONT STUFF */
+
+.intro_title {
+  color: #fff;
+  position: relative;
+  left: 770px;
+  top: 140px;
+  font-size: 800%;
+  line-height: 90px;
+  font-family: 'Cormorant SC', serif;
+  text-shadow: 3px 3px #999;
+}
+.sub-heading {
+  color: #fff;
+  position: relative;
+  left: 800px;
+  top: 150px;
+  font-size: 200%;
+}
+
+#page2-heading {
+  color: #fff;
+  font-size: 220%;
+  columns: 500px;
+  padding: 40px;
+}
+
+.page2-content {
+  padding: 30px;
+  -webkit-column-break-inside: avoid;
+  color: #fff;
+  font-size: 150%;
+  column-width: 450px;
+  width: 50%;
+  text-shadow: 2px 2px black;
+}
+
+
+.parallax_layer-base {
+  padding: 30px;
+  column-break-inside: avoid;
+}
+.book, .year {
+  position: relative;
+  text-align: center;
+  z-index: 999;
+  color: #fff;
+  text-shadow: 2px 2px #000;
+  line-height: 9vh;
+}
+.year{
+  font-size: 40%;
+}
+
+.book {
+  font-size: 295%;
+}
+
+#page3-heading {
+  color: white;
+  text-align: center;
+  font-size: 180%;
+  line-height: 95px
+}
+
+a, div{
+  font-family: 'EB Garamond', serif;
+}
+
+
+
+/* NAV MENU */
+
+#menu {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 9999;
+  width: 100%;
+  height: 50px;
+  background-color: #333;
+  opacity: 0.8;
+}
+--------------------
```

Comment:
Watch out for lines like this that should be commented out; they might stop the CSS rules that follow from working.